cmd/nri-kubernetes/main.go

```diff
@@ -68,7 +70,9 @@ func main() {
 		logger.SetLevel(log.DebugLevel)
 	}
 
-	i, err := createIntegrationWithHTTPSink(c)
+	cache := storer.NewInMemoryStore(storer.DefaultTTL, storer.DefaultInterval, logger)
+
+	i, err := createIntegrationWithHTTPSink(c, cache)
```

Comment:
Could we create the custom storer inside `createIntegrationWithHTTPSink`, rather than passing it? We could then save the extra parameter to `createIntegrationWithHTTPSink`.